community/kernel/src/main/java/org/neo4j/kernel/impl/store/format/StoreVersion.java

```diff
@@ -24,25 +24,32 @@
  */
 public enum StoreVersion
 {
-    STANDARD_V2_0( "v0.A.1" ),
-    STANDARD_V2_1( "v0.A.3" ),
-    STANDARD_V2_2( "v0.A.5" ),
-    STANDARD_V2_3( "v0.A.6" ),
-    STANDARD_V3_0( "v0.A.7" ),
+    STANDARD_V2_0( "v0.A.1", "2.0.0" ),
+    STANDARD_V2_1( "v0.A.3", "2.1.0" ),
+    STANDARD_V2_2( "v0.A.5", "2.2.0" ),
+    STANDARD_V2_3( "v0.A.6", "2.3.0" ),
+    STANDARD_V3_0( "v0.A.7", "3.0.0" ),
 
-    HIGH_LIMIT_V3_0_0( "vE.H.0" ),
-    HIGH_LIMIT_V3_0_6( "vE.H.0b" ),
-    HIGH_LIMIT_V3_1_0( "vE.H.2" );
+    HIGH_LIMIT_V3_0_0( "vE.H.0", "3.0.0" ),
+    HIGH_LIMIT_V3_0_6( "vE.H.0b", "3.0.6" ),
+    HIGH_LIMIT_V3_1_0( "vE.H.2", "3.1.0" );
 
     private final String versionString;
+    private final String firstNeo4jVersion;
 
-    StoreVersion( String versionString )
+    StoreVersion( String versionString, String firstNeo4jVersion )
     {
         this.versionString = versionString;
+        this.firstNeo4jVersion = firstNeo4jVersion;
     }
 
     public String versionString()
     {
         return versionString;
     }
+
+    public String firstNeo4jVersion()
+    {
+        return firstNeo4jVersion;
```

Comment:
i would really like if we will call this version `introductionVersion` or `introducedInVersion`. First version is not descriptive by itself and as soon as we use `first` it mean that we have some order in mind which is not clear.